studio/src/ui/popoverMenus/PopoverManager.js

```diff
@@ -119,17 +117,23 @@ export class PopoverManager {
 	}
 
 	#hasBodyClickListener = false;
+
 	/**
 	 * @param {MouseEvent} e
 	 */
 	#onBodyClick = e => {
-		if (this.current && !this.current.el.contains(/** @type {Node} */ (e.target))) {
```

Comment:
Seems like the removal of this line causes popovers to close when clicking inside the popovers themselves as well now. Actually I think that's why the one e2e test might be failing as well. I ran it a bunch of times but it keeps failing. It just happens to *also* be flaky as well.

Considering that it's now possible for multiple popovers to be open. I think the best behaviour would be to not close any popovers at all when any of them is clicked. Only the rest of the body/curtain should close popovers.